src/components/configdrop/ConfigDrop.test.tsx

```diff
@@ -0,0 +1,53 @@
+import * as enzyme from "enzyme";
+import { DropzoneArea } from 'material-ui-dropzone';
+import * as React from "react";
+import { ClientConfig } from '../../models/client_config';
+import { ConfigDrop } from "./ConfigDrop";
+
+describe("ConfigDrop", () => {
+    const verifyConfig = (configObj: ClientConfig) => {
```

Comment:
There is a potential problem here where the test will pass if this function isn't called. Take a look at Banner.test.tsx for how you can ensure that it's called